src/masoniteorm/exceptions.py

```diff
@@ -1,2 +1,10 @@
 class DriverNotFound(Exception):
     pass
+
+
+class ModelNotFound(Exception):
+    pass
+
+
+class HTTP404(Exception):
```

Comment:
Here we start to couple this independant package with http stuff... I really like this `find_or_404` method though (coming from Django !).
In addition, such an exception should be provided by Masonite core itself, isn't it ? 

But I don't really see know how to deal with all that, I guess it could be let as is and improved later ?
@Marlysson @josephmancuso what do you think ?

Reply:
for Masonite we have "ExceptionHandlers". We already use this feature for other things. We return different views based on the exceptions thrown. 

For example if you have ever used the `dd` helper, that throws a `DumpException`, we catch he exception and show a new view (that debug screen).

So it honestly doesn't matter what we name this exception, as long as its unique enough that frameworks are able to catch it

Reply:
So in this example we will need a new exception handler for HTTP404

Reply:
Okay, makes sense.

Reply:
Yeah, this make sense. So when use orm as standalone project should be this method unavailable?

Reply:
yeah it wouldn't make sense for someone to use this method if their framework didn't handle it. I'm 99% sure flask handles custom exceptions. I know I used exceptionite on a flask project which takes a similar approach to Masonite. 

So if someone used this with Flask they'd have to register the HTTP404 error somehow